### DISCUSSION TITLE
Update the explore section more frequently

### DISCUSSION THREAD

**Discussion by vinhig** [community/community#6042] [Mobile] [answered]
Sometimes, you open your GitHub app just to check what's new/trending among the developers community. Sadly, the explore section is rarely updated. I mean, featured repo changes daily afaik. But the personalized section (which is quite interesting ngl) exposes new things weekly... Which is not enough in my opinion. I want new repos to discover more frequently!

**Comment by taki-on** [comment, marked as answer]
Thank you for your suggestion! We will discuss this in the team :)
